app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -212,6 +212,13 @@ def finish(*_args)
 
   alias_method :abort_job, :cleanup
 
+  def queue_callback(state, msg, result)
+    _log.info("queue_callback called with pod name #{state}, #{msg}, #{result}")
```

Comment:
we probably don't want this logging each time but only when the signal is actually queued